keyboards/dtd/ldmlKeyboard.xsd

```diff
@@ -299,7 +299,7 @@
         <xs:element minOccurs="0" maxOccurs="unbounded" ref="special"/>
       </xs:sequence>
       <xs:attribute name="id" type="xs:NMTOKEN"/>
-      <xs:attribute name="modifier" type="xs:NMTOKEN"/>
+      <xs:attribute name="modifier" type="xs:NMTOKENS"/>
```

Comment:
Can we list the possible modifiers here?

Reply:
probably not *here* right now, but will list them in annotations

Reply:
CLDR-15990 is the general ticket for dtd -> xsd and covers converting the @ -annotations mapping to more precise XSD.